rotkehlchen/chain/ethereum/node_inquirer.py

```diff
@@ -263,11 +265,27 @@ def assure_curve_protocol_cache_is_queried(self) -> bool:
             return False
 
         curve_address_provider = self.contracts.contract('CURVE_ADDRESS_PROVIDER')
-        pools = query_curve_registry_pools(ethereum=self, curve_address_provider=curve_address_provider)  # noqa: E501
+        get_registry_result = curve_address_provider.call(
```

Comment:
Thinking of this shouldn't you also add a DB migration here for this? To make sure that curve gauges are queried? Otherwise a user who was running 1.27.1 will not see any gauges data until `should_update_protocol_cache(GeneralCacheType.CURVE_LP_TOKENS)` is `True` again.

Perhaps just resetting last update ts for all of them would do it?

Or another approach is to add a new `should_update_protocol_cache(GeneralCacheType.CURVE_GAUGES)` variable, which would then not be set and as such need no migration.

I guess updating them separately is not needed so perhaps just adding a small deletion of last_update in the migration is best.